org.eclipse.lsp4e/src/org/eclipse/lsp4e/progress/LSPProgressManager.java

```diff
@@ -149,10 +157,15 @@ private void report(final WorkDoneProgressReport report, final IProgressMonitor
 	 *            the {@link ProgressParams} used for the progress notification
 	 */
 	public void notifyProgress(final @NonNull ProgressParams params) {
-		String jobName = params.getToken().getLeft();
-		BlockingQueue<ProgressParams> progress = progressMap.get(jobName);
+		String jobIdentifier = params.getToken().getLeft();

```

Comment:
```suggestion
		String jobIdentifier = params.getToken().map(Function.identity(), Object::toString);
```
and also on line 61.

Reply:
That is nice, I did not know that API. Thanks.